config/locales/ca.yml

```diff
@@ -345,6 +349,25 @@ ca:
       text: "%{organization_name} en"
       text_donation: L'associació ADBdT ofereix TimeOverflow de forma gratuïta als Bancs de Temps. Amb una donació de %{href} pots ajudar a mantenir i millorar la plataforma.
       text_donation_link: 1€ al mes
+  multi_transfers:
+    multi_transfers: 'Transferències múltiples '
+    step:
+      confirm:
+        description: Confirma els canvis
+        type_of_transfer: Tipus de transferència
+      select_type:
+        description: Selecciona  el tipus de transferència
```

Comment:
There are 2 spaces between `Selecciona` and `el`, right?

Reply:
OCD :+1: I will fix it